packages/adminrouter/extra/src/docs/api/nginx.agent.yaml

```diff
@@ -17,6 +20,10 @@ backends:
     name: DC/OS Component Package Manager (Pkgpanda)
     reference: 'https://dcos.io/docs/1.10/administering-clusters/component-management/'
 routes:
+  /:
```

Comment:
Is this an oversight?

Reply:
I generated this commit's contents, including this added backend, by running `make api-docs` in `packages/adminrouter/extra/src/`. Should I remove this stanza anyway?

Reply:
CC @vespian @ahairape

Reply:
Ping @vespian @ahairape

Reply:
Apparently, it is a valid representation corresponding to the _root_ directive found in NGINX .conf files. Since there is no _root_ directive in **_nginx.agent.conf_** (or other .conf files included from it),
```
group: Other
matcher: path
path: /
```
stanza is generated.

The **_nginx.master.conf_**, on the other hand, includes the config file _includes/server/master.conf_ which has the following _root_ directive:
```
# Group: Root
# Description: DC/OS GUI
root /opt/mesosphere/active/dcos-ui/usr;
```

The corresponding stanza is 
```
/:
    group: Root
    matcher: path
    description: DC/OS GUI
    file: /opt/mesosphere/active/dcos-ui/usr
    path: /
```

Reply:
The API docs are not regenerated as often as they should. This change stems from the bugfix I did while fixing some other issue in is some other PR - it was not accompanied by the API docs re-generation. There is a CI check that should be checking it but I suppose it no longer works :(

Anyway - this is desired, thanks @branden for re-generating API docs :ship: